project/AppDependencies.scala

```diff
@@ -3,11 +3,12 @@ import sbt.librarymanagement.InclExclRule
 
 object AppDependencies {
 
-  private val bootstrapVersion = "9.5.0"
+  private val bootstrapVersion = "9.6.0"
 
   val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"             %% "bootstrap-backend-play-30"  % bootstrapVersion,
-    "org.apache.xmlgraphics"  %  "fop"                        % "2.9",
+    "commons-io"              %  "commons-io"                 % "2.18.0",
```

Comment:
Is this new dependency needed?

Reply:
I see why now 🙂 https://catalogue.tax.service.gov.uk/vulnerabilities?vulnerability=CVE-2024-47554

We can pop this dependency in the overrides instead

Reply:
Using sbt dependencyTree it looks like bumping the fop dependency 2.10 resolves https://catalogue.tax.service.gov.uk/vulnerabilities?vulnerability=CVE-2024-28168 so all good with that one 👍